lms/djangoapps/instructor_task/tests/test_migrations.py

```diff
@@ -0,0 +1,61 @@
+"""
+Test migrations instructor_task table.
+"""
+# pylint: disable=attribute-defined-outside-init
```

Comment:
Can you verify why we need this? 
Which attribute definition causing pylint to fail?